jahnvisahni31/DesignDeck
# FAQ section Design

### Pull Request: Implement FAQ Section for Improved SEO

**Related Issue:** #172 

#### Description
This PR introduces a new FAQ section designed to enhance SEO performance. The implementation is fully responsive and interactive, ensuring a seamless user experience across devices. Additionally, the FAQ section supports both dark and light mode to accommodate user preferences.

#### Changes Made
- Added a responsive FAQ section.
- Implemented interactive features (e.g., expandable/collapsible answers).
- Integrated light and dark mode support.
- Optimized for SEO with structured data markup.

#### How to Test
1. Pull the latest changes.
2. Check the FAQ section on various devices (mobile, tablet, desktop).
3. Toggle between dark and light mode to ensure proper functionality.
4. Verify SEO improvements through a structured data testing tool.


#### Additional Notes
- Please review the responsiveness on different screen sizes.
- Feedback on SEO optimization is welcome.
